integrations/sensu/entity-manager/sensu-integration.yaml

```diff
@@ -0,0 +1,58 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: sensu
+  name: entity-manager
+spec:
+  class: supported
+  provider: discovery
+  display_name: "Sensu Entity Manager"
+  short_description: "Real-time subscription and entity metadata management (for use with auto-discovery checks)"
+  supported_platforms:
+    - darwin
+    - linux
+    - windows
+  tags:
+    - sensu
+    - discovery
+    - auto discovery
+    - service discovery
+  contributors:
+    - "@sensu"
+    - "@calebhailey"
+  prompts:
+    - type: section
```

Comment:
Need to convert this section to using the optional secrets pattern like in elasticsearch integration